File: dotnet/Src/UserService/FakeDb.cs
using System;
using System.Collections.Generic;
using UserService.Model;

namespace UserService
{
    internal static class FakeDb
    {
        internal static IEnumerable<User> Users = 
            new List<User>()
            {
                new User(1, "gary.oldman@example.com", "Gary", "Oldman"),
                new User(2, "jack.nicholson@example.com", "Jack", "Nicholson")
            };
    }
}

File: scala/src/main/scala/users/db/FakeDb.scala
package users.db

import users.model.User

object FakeDb {
  val Users: Seq[User] = Seq(
    User(1, "gary.oldman@example.com", "Gary", "Oldman"),
    User(2, "jack.nicholson@example.com", "Jack", "Nicholson")
  )
}
